build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
```

Comment:
Just a heads up that I am not keen on switching from rake to bash. Nothing against your bash of course, which looks great. In general, though, as bash scripts grow in complexity I find them very difficult to read. I've found this to be true, even when I have written them myself.

Also, on a personal note, I work infrequently enough on these projects that it's extra important to document all the project setup setups. Another reason I like rake, is that it's easy to write lots of one-liners for the random bits, and run `rake -T` for a quick reminder.